runner/cf-driver/run.sh

```diff
@@ -7,9 +7,13 @@ source "${currentDir}/base.sh"
 
 printf "[cf-driver] Using SSH to connect to %s and run '%s' step\n" "$CONTAINER_ID" "$2"
 
-# Add line below script shebang to source the /etc/profile
-sed -i '2isource /etc/environment\n' "$1"
-sed -i '2isource /etc/profile\n' "$1"
+# Add line below script's shebang to source
+# /etc/profile, etc/environment & the $HOME/bin
+sed -i '' '1a\
```

Comment:
Has this been tested in a deployed environment? I read something recently that the `sed` on all (some?) linux distros doesn't deal well with `-i ''`

Reply:
Hm, I thought I did but I'll double check to be sure. I'd put it there because I'd thought it would do the opposite and maximize compatibility :-) will look further into it in any case.

Reply:
You were right! seems like it's best to just avoid `-i` altogether if you're aiming for portability. Thanks for catching.